src/components/discovery/LocusSearch.js

```diff
@@ -59,9 +74,10 @@ const LocusSearch = ({assemblyId, addVariantSearchValues}) => {
             return;
         }
 
-    // don't use locus.assemblyId, since this is the lookup value
         const {chrom, start, end} = locus;
         addVariantSearchValues({chrom: chrom, start: start, end: end});
```

Comment:
this can be shortened to `{chrom, start, end}` - and the same object is used for `setCurrentLocus` as well

i mention this again below, but this "make reduced locus form from locus" pattern appears twice and could probably be factored out; that function could describe why only these fields are included in this version of the locus object.